napari/utils/info.py

```diff
@@ -1,8 +1,73 @@
+import os
 import platform
+import subprocess
 import sys
 
 import napari
 
+OS_RELEASE_PATH = "/etc/os-release"
+
+
+def _linux_sys_name():
+    """
+    Try to discover linux system name base on /etc/os-release file or lsb_release command output
+    https://www.freedesktop.org/software/systemd/man/os-release.html
+    """
+    if os.path.exists(OS_RELEASE_PATH):
+        with open(OS_RELEASE_PATH) as f_p:
+            data = {}
+            for line in f_p:
+                field, value = line.split("=")
+                data[field.strip()] = value.strip().strip('"')
+        if "PRETTY_NAME" in data:
+            return data["PRETTY_NAME"]
+        if "NAME" in data:
+            if "VERSION" in data:
+                return f'{data["NAME"]} {data["VERSION"]}'
+            if "VERSION_ID" in data:
+                return f'{data["NAME"]} {data["VERSION_ID"]}'
+            return f'{data["NAME"]} (no version)'
+
+    try:
+        res = subprocess.run(
+            ["lsb_release", "-d", "-r"], check=True, capture_output=True
+        )
+        text = res.stdout.decode()
+        data = {}
+        for line in text.split("\n"):
+            key, val = line.split(":")
+            data[key.strip()] = val.strip()
+        version_str = data["Description"]
+        if not version_str.endswith(data["Release"]):
+            version_str += " " + data["Release"]
+        return version_str
+    except subprocess.CalledProcessError:
```

Comment:
should we maybe make this a bare `Exception`?  is there possible exception here that we wouldn't want to just pass?

Reply:
I prefer avoiding bare exceptions — at least, if something else crashes here, it reflects that we don't know something, and we should probably change our code to reflect how we handle it, rather than suppress it forever.